examples/nemo/scripts/utils.py

```diff
@@ -68,24 +65,41 @@
                     # prevents from being in-lined (and then breaks any attempt
                     # to create OpenACC regions with calls to them)
                     "interp1", "interp2", "interp3", "integ_spline", "sbc_dcy",
-                    "sum", "sign_", "ddpdd"]
+                    "sum", "sign_", "ddpdd", "psyclone_cmp_int",
+                    "psyclone_cmp_char", "psyclone_cmp_logical"]
 
 # Currently fparser has no way of distinguishing array accesses from
 # function calls if the symbol is imported from some other module.
 # We therefore work-around this by keeping a list of known NEMO functions.
 NEMO_FUNCTIONS = ["alpha_charn", "cd_neutral_10m", "cpl_freq", "cp_air",
-                  "eos_pt_from_ct", "gamma_moist", "l_vap",
+                  "eos_pt_from_ct", "gamma_moist", "l_vap", "q_air_rh",
                   "sbc_dcy", "solfrac", "psi_h", "psi_m", "psi_m_coare",
                   "psi_h_coare", "psi_m_ecmwf", "psi_h_ecmwf", "q_sat",
                   "rho_air", "visc_air", "sbc_dcy", "glob_sum",
                   "glob_sum_full", "ptr_sj", "ptr_sjk", "interp1", "interp2",
-                  "interp3", "integ_spline"]
+                  "interp3", "integ_spline", "nf90_put_var"]
 
 # Currently fparser has no way of distinguishing array accesses from statement
 # functions, the following subroutines contains known statement functions
 CONTAINS_STMT_FUNCTIONS = ["sbc_dcy"]
 
-VERBOSE = False
+# These files change the results from baseline when psyclone processes them
+PASSTHROUGH_ISSUES = [
+    "ldfslp.f90",  # It has a '!dir$ NOVECTOR' that gets deleted by fparser
+]
+
+# These files change the results from the baseline when psyclone adds
+# parallelisation dirctives
+PARALLELISATION_ISSUES = [
+    "ldfc1d_c2d.f90",
+    "tramle.f90",
+    # These files get the same results when parallelised by PSyclone and using
+    # conservative opt flags (e.g. nvfortran -O1 -Kieee -nofma -Mnovect) but
+    # can be excluded for more stable results when using more aggressive flags
+    "icedyn_rhg_evp.f90",
```

Comment:
Maybe, but ideally I want to fix the issues, I just felt that this PR is already doing too much, but I could explore it in the follow up one which I added a TODO for. Note that needs to be fixed for the GPU because it has a big impact there.

If they cannot be solved, we could explore being enabled by a transformation parameter, but I don't like the idea of "knowingly letting race conditions pass because it gives better performance".